### ISSUE TITLE
MacOS 12.1 browseForSave ignores the default file name

### ISSUE THREAD

**Issue by myflashlab** [airsdk/Adobe-Runtime-Support#1529]
AIR SDK: 33.1.1.731

When using the File class `browseForSave`, I need to specify a default name and extension for the new file to be saved for example:

```actionscript
var file:File = dis.parent.resolvePath($model.name + ".keystore");

var f:File = File.desktopDirectory.resolvePath(file.name);
f.addEventListener(Event.SELECT, onSelected);
f.addEventListener(Event.CANCEL, onCanceled);
f.browseForSave("save keystore");
```
In older SDKs, this opens the save window and sets the default file name to be saved to something like `name.keystore` but since Mac 12.1 and the whole problem around the native browse window, This functionality is lost. and no matter what the value for the `file` object is, it will always show `Untitled` instead of name+extension.

**Comment by Danik** [comment]
I also ran into this problem with 33.1.731. With 33.1.1.713 it behaves as expected.

**Comment by myflashlab** [comment]
This is now fixed when updating to 31.1.1.743
